src/services/feed_validator.py

```
@@ -2,30 +2,31 @@
         """Initialize the feed validator.
-        
+
         Args:
             timeout (int): Maximum time in seconds to wait for a feed response
             retry_delay (float): Delay in seconds between validation attempts
         """
         self.timeout = ClientTimeout(total=timeout)
         self.retry_delay = retry_delay
         self.headers = {
-            'User-Agent': 'OhPeehMel/1.0 (https://github.com/yourusername/ohpeehmel; feed-validator) Python-Feedparser/6.0.11'
+            "User-Agent": "OhPeehMel/1.0 (https://github.com/yourusername/ohpeehmel; feed-validator) Python-Feedparser/6.0.11"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)